tests/ra_aid/tools/test_execution.py

```
@@ -0,0 +1,218 @@
+        "original prompt",
+        0,
+        True,
+        {
+            "shell_cmd_result_error": Exception("Command failed")
+        },
+        (True, "original prompt", True, 1)
+    ),
+]
+
+@pytest.mark.parametrize(
+    "name,config,original_prompt,test_attempts,auto_test,mock_responses,expected",
```

Ruff (0.8.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)